### DISCUSSION TITLE
How to change wifi network?

### DISCUSSION THREAD

**Discussion by thitcher** [softwarecrash/EPEver2MQTT#57] [Q&A] [answered]
Hi,
want to switch over to my IoT network, but can´t find the opinion to change the wifi.
Can somebody tell me how to switch over to another wifi over the EPEver2MQTT webinterface?

**Comment by softwarecrash** [comment]
https://github.com/softwarecrash/EPEver2MQTT/wiki/Reset

**Reply by thitcher** [reply]
Thanks for the answer, I´ve allready read that, but I was not sure if reset means "reset of network settings" or "reset off all settings" (including the Firmware) which will need to connect the esp over wire to flash again.
So what kind of reset will it be?

**Reply by softwarecrash** [reply, marked as answer]
> Thanks for the answer, I´ve allready read that, but I was not sure if reset means "reset of network settings" or "reset off all settings" (including the Firmware) which will need to connect the esp over wire to flash again. So what kind of reset will it be?

it will reset all settings, the firmware itself canot be deleted by itself.
thats the only way to change wifi settings at the moment, or bring the esp out of range (or turn off) known wifi, so the esp will bring back the config AP

**Reply by thitcher** [reply]
Thanks a lot!
Will disable the actual wifi and re-connect it with the new network.
